lzero/worker/muzero_collector.py

```diff
@@ -490,12 +520,21 @@ def collect(self,
                 if timestep.done:
                     self._total_episode_count += 1
                     reward = timestep.info['eval_episode_return']
-                    info = {
-                        'reward': reward,
-                        'time': self._env_info[env_id]['time'],
-                        'step': self._env_info[env_id]['step'],
-                        'visit_entropy': visit_entropies_lst[env_id] / eps_steps_lst[env_id],
-                    }
+                    if self.policy_config.gumbel_algo:
```

Comment:
first create four common variables, then append the one related to gumbel muzero